src/main/java/org/cryptomator/cli/Args.java

```diff
@@ -133,4 +114,26 @@ public static void printUsage() {
 		new HelpFormatter().printHelp(USAGE, OPTIONS);
 	}
 
+	public PasswordStrategy addPasswortStrategy(final String vaultName) {
```

Comment:
Not sure if we actually need this method. I am aware it is used in `CryptomatorCli.validate` but in fact think we should move the whole validation stuff into the Args class. `Args.parse` should fail with exceptions if the provided args are invalid.

Nothing to be done in the related PR though, so OK for now.